packages/controller/src/main.ts

```diff
@@ -2889,12 +2890,14 @@ async function processMessage(msg: ioBroker.SendableMessage): Promise<null | voi
                         }
                     }
 
-                    if (!error) {
+                    if (!error && config) {
                         try {
                             fs.writeFileSync(configFile, JSON.stringify(config, null, 2));
                         } catch {
                             error = `Cannot write file ${configFile}`;
                         }
+                    } else if (!error) {
+                        error = `Invalid data for writeBaseSettings ${msg.from}`;
```

Comment:
Hm if we don't have an error yet we will always assign one here.. and the invalid data logged is the msg.from attribute? I think something is wrong here

Reply:
Hm okay the logic is okay as it only happens if config is falsy but shouldn't we log the received config then